patterns/cell-based-eks/1.cell1/variables.tf

```diff
@@ -0,0 +1,12 @@
+
+variable "name" {
```

Comment:
we don't provide variables unless we absolutely require something from users to deploy (i.e. - a domain name or hosted zone) - these are not consumed in place, they are references

Reply:
Removed the references